ramls/types/tags/tagCollection.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Package Collection Schema",
+  "description": "Tag Collection Schema",
+  "javaType": "org.folio.rest.jaxrs.model.TagCollection",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "data": {
+      "type": "array",
+      "description": "List of tags",
+      "items": {
+        "type": "string"
```

Comment:
We might want to check with the UI team if this will break if we do not have "id", "attributes" objects etc. in "data" like in here fro example: https://github.com/folio-org/mod-kb-ebsco-java/blob/master/ramls/types/packages/packageCollectionItem.json

Reply:
will be refactored

Reply:
👍